utils/validation/scripts/keywords_validation.py

```diff
@@ -0,0 +1,30 @@
+"""Check if the list of keywords in validation/ matches the one from the glossary."""
```

Comment:
I'm not sure this script is needed, utils/validation/scripts/validation.py is doing something similar already and you can now execute it as a separate script with python utils/validation/scripts/validation.py --file experiments/my-experiment.yaml
It checks against keywords present in utils/validation/keywords.yaml only but that should be enough I believe.

Reply:
This is more from a documentation POV, to make sure that all keywords that are the `utils/validation/keywords.yaml` list are indeed explained and defined in the documentation. 
But if it's not needed we can remove it later on.

Reply:
I understand, maybe we run it in the github actions to make sure the documentation is always in sync before merge. I'll try to do that later and maybe do some renaming of the files in utils/validation/scripts to make them a bit more descriptive